package.json

```diff
@@ -31,6 +32,6 @@
     "underscore": "^1.7.0"
   },
   "engines": {
-    "node": "0.10.x"
+    "node": "4.2.x"
```

Comment:
DId you try running Hubot locally to verify that this works? `hubot` may need to be upgraded.